tests/unit/lib/NavLink-test.js

```diff
@@ -397,19 +397,4 @@ describe('NavLink', function () {
             }, 50);
         });
     });
-
-    describe('componentWillUnmount', function () {
-        it('should update active state', function () {
-            var div = document.createElement('div');
```

Comment:
Any reason this test was removed? I think the test was misnamed, but should still be valid right?